src/core/type.ml

```diff
@@ -2017,6 +2017,9 @@ let rec unify a b =
 	| TAbstract ({a_path=[],"Void"},_) , _
 	| _ , TAbstract ({a_path=[],"Void"},_) ->
 		error [cannot_unify a b]
+    | TAbstract ({ a_path = ["haxe"],"NotVoid" },[]), _
```

Comment:
Please, change spaces to tabs as in the neighbring lines.

Reply:
O_o sorry, seems like my editor is not configured correctly on this computer..